mitosheet/src/components/elements/RadioButtonBox.tsx

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Mito
+import React from 'react';
+import { classNames } from '../../utils/classNames';
+import '../../../css/elements/RadioButtonBox.css'
+import { Height, Width } from './sizes.d';
+import RadioButtonItem from './RadioButtonItem';
+
+
+interface RadioButtonsProps {
+    /**
+        * @param values - The options for the user to pick between 
+    */
+    values: string[];
```

Comment:
I think we want this to have the same interface as the MultiToggleBox. Generally, making the creator of a radio button responsible for creating the RadioButtonItems is a much better call - as in this current architecture, it gets very complicated to have the displayed text different than the id that is displayed.

So, this should take children, and no selected value. The onChange can either be defined on the RadioButton (and then copied to the child - check out the MultiToggleBox `React.cloneElement`) - or it could be on each RadioButtonItem.

Reply:
I just did it this way because I didn't want to over architect the radio button for the current use case. We don't need anything fancy now.

Reply:
Fair enough!